CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - Refactor s3 submodule to minimize resource usage (PR [#569](https://github.com/RaRe-Technologies/smart_open/pull/569), [@mpenkov](https://github.com/mpenkov))
+- Change download_as_bytes to download_as_string in gcs submodule (PR [#571](https://github.com/RaRe-Technologies/smart_open/pull/571), [@alexandreyc](https://github.com/alexandreyc))
```

Comment:
Should reverse `download_as_bytes` and `download_as_string` :)

Reply:
Whoops, my bad.

Reply:
I just spotted this too, in the release notes:

```
Migrate download_as_string to download_as_bytes in gcs submodule (PR [#571](https://github.com/RaRe-Technologies/smart_open/pull/571), [@alexandreyc](https://github.com/alexandreyc))
```

Reply:
I fixed it in the release notes, but it's too late for the package on PyPI.

Reply:
I don't think this affect the package on PyPI. The only issue (a very minor one) is with CHANGELOG + release notes.

Reply:
No, it's purely cosmetic.